src/components/Logsheet/Logsheet.jsx

```diff
@@ -154,6 +119,43 @@ function rowContent(_index, row) {
 }
 
 function Logsheet() {
+    const [messages, setMessages] = useState([
```

Comment:
Added this sample data back here to display on our screens in the case the database is not set up. If connection is active it will overwrite the data.

Reply:
good idea